cardano-testnet/src/Test/Base.hs

```diff
@@ -1,12 +1,20 @@
 module Test.Base
   ( integration
+  , isLinux
   ) where
 
+import           Data.Bool (Bool)
+import           Data.Eq (Eq (..))
 import           Data.Function
 import           GHC.Stack (HasCallStack)
+import           System.Info (os)
 
 import qualified Hedgehog as H
 import qualified Hedgehog.Extras.Test.Base as H
 
+
```

Comment:
We need to decide which we are going to keep. `Test.Base` or `Test.Util`. If you don't want to do this in this PR, can you create an issue to track it?

Reply:
https://github.com/input-output-hk/cardano-node/issues/4148